test/integration/bootstrap/bootstrap_test.go

```diff
@@ -34,6 +40,30 @@ func getResultFieldStrSlice(rs []gjson.Result, field string) []string {
 	return s
 }
 
+func checkAPIEnabled(t *testing.T, projectID, api string) (string, error) {
```

Comment:
Any reason why we need the string?
```suggestion
func checkAPIEnabled(t *testing.T, projectID, api string) (error) {
```

Reply:
the return `(string, error)` was part of the signature of `retry.DoWithRetry`.
changed to `(bool, error)` because of the signature of `utils.Poll`